pages/instructorAssessmentGroups/csvHelpInstanceGroups.ejs

```diff
@@ -0,0 +1,18 @@
+<p>
+  Upload a CSV file in the format of
+  the <tt><i>&lt;assessment&gt;</i>_grouping.csv</tt>
```

Comment:
This filename doesn't match the name of the file downloaded from the instructor downloads page